mongock-spring-v5/src/main/java/com/github/cloudyrock/spring/v5/MongockTestContext.java

```diff
@@ -1,41 +0,0 @@
-package com.github.cloudyrock.spring.v5;
-
-import com.github.cloudyrock.mongock.MongockConnectionDriver;
-import org.bson.BsonDocument;
-import org.springframework.beans.factory.InitializingBean;
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
-import org.springframework.context.annotation.Import;
-import org.springframework.data.mongodb.core.MongoTemplate;
-
-@Configuration
-@Import(MongockCoreContextSelector.class)
-public class MongockTestContext {
-
-
-  @Bean
-  public MongockTestDriverInitializingBean mongockTestDriverInitializingBean(MongockConnectionDriver connectionDriver,
-                                                                             MongoTemplate mongoTemplate) {
-    return new MongockTestDriverInitializingBean(connectionDriver, mongoTemplate);
-  }
-
-
-  public static class MongockTestDriverInitializingBean implements InitializingBean {
-
-    private final MongockConnectionDriver driver;
-    private final MongoTemplate mongoTemplate;
-
-    private MongockTestDriverInitializingBean(MongockConnectionDriver driver,
-                                              MongoTemplate mongoTemplate) {
-      this.driver = driver;
-      this.mongoTemplate = mongoTemplate;
-    }
-
-    @Override
```

Comment:
Why is this removed?

Reply:
It was unused. If we want to keep it, I believe it should be part of the test classpath.

Reply:
It's requires in some test scenario, when they want Mongock to be enabled but still want to call the changeSet directly. It's an edge case for a user, but still relevant. I rather keep it ;)